client/src/pages/hostSetup.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from "../styles/hostSetup.module.css";
+import passwordStyle from "../styles/Textbox.module.css";
+import Textbox from "../components/Textbox";
+import Checkbox from "../components/Checkbox";
+import Button from "../components/Button";
+const hostSetup = () => {
+    return (
+        <div className={styles.main}>
+            <div className={styles.title}>Host a Room</div>
+            <div className={styles.grid}>
+                <Textbox placeholder="Room Name" />
+                <Checkbox text="Only show games that everyone has" />
+                <div className={styles.enableAuth}>
+                    <div className={styles.authSpacing}>
+                        <Checkbox text="Enable Authentication" />
+                    </div>
+                    <input className={passwordStyle.main} type="password" />
```

Comment:
Best way to do this is to make `Textbox` an `<input type="text">` instead of a `<textarea>` (or add a new component like `TextInput`) and add a property like `isPassword` that makes it a `<input type="password">`. I'd say avoid importing styles from component-specific CSS modules in general since its a recipe for spaghetti.

Reply:
I have done this! 🤙